lib/SPIRV/libSPIRV/spirv.hpp

```diff
@@ -1,4 +1,3 @@
-// clang-format off
```

Comment:
@MrSidims if I understand correctly this is a clean file from the SPIRV-Headers project? ("clean" in the sense of the current version + your 2 pending SPIRV-Headers PRs, so no translator-specific changes were made)
If so, @AlexeySachkov perhaps we should add this file to the ignore list for the formatting checks?

Reply:
Yes, it is. It was generated via ./bin/makeHeaders tool, so I think we may ignore the checks.
There are few extensions missing still, I'll remove WIP when it's ready.

Reply:
Okay. Given that you're removing spirv.hpp in #813 there is no value in updating the checks to ignore the file now, so we can just ignore the formatting issues in this PR.